### PR TITLE
Client README.md file update

### PR DESCRIPTION
**Pull request by akihikokuroda**
### Summary
Update the out dated qiskit_serverless client README file

### CODE REVIEW

File: client/README.md [hunk 1]
```diff
@@ -21,105 +21,100 @@ Full docs can be found at https://qiskit-extensions.github.io/qiskit-serverless/
...
-       arguments={
-           "circuits": circuits
-       }
-   )
+from qiskit_serverless import ServerlessClient, QiskitFunction
+from qiskit.circuit.random import random_circuit
+
+client = ServerlessClient(
+    token="awesome_token", 
+    host="http://localhost:59994",
```
Comment (Tansito) [inline client/README.md:77]: Is it port `599994` the default one, @akihikokuroda ? Just to confirm 👍

**Review by Tansito — APPROVED** [review]
LGTM, thank you @akihikokuroda . I just looked for a double check about the port 👍

**Comment by psschwei** [comment]
One thought - should we remove the examples from this doc and just link to the ones in the `docs/` folder? That way we only have one place we need to update when we do our quarterly refactoring...

File: client/README.md (hunk 1 above)
Comment (akihikokuroda) [reply client/README.md:77]: oops.  I was going to change them before checkin.  I'll fix it.

**Comment by akihikokuroda** [comment]
@psschwei That seems a good idea.   Or we may want to put a short migration guide here with the samples when we change the api.

**Pull request merged**